metricbeat/module/kibana/actions/_meta/fields.yml

```diff
@@ -0,0 +1,27 @@
+- name: actions
+  type: group
+  description: >
+    Kibana actions metrics.
+  release: beta
+  fields:
+    - name: kibana
+      type: group
+      fields:
+        - name: status
```

Comment:
This field is gonna be located in `kibana.actions.kibana.status`. If that is your expected path it's fine but maybe you want to remove the second `kibana` for clarity and leave it in `kibana.actions.status`.

In such case just remove the upper level `kibana` entry on line 7